containers/roles/development/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+  # Get facts such as database `adapter` and `uid` for the next tasks.
+- include_tasks: facts.yml
+  tags:
+    - always
+
+- name: docker-build image
+  docker_image:
+    name: "{{image_name}}"
+    tag: "{{image_tag}}"
+    path: images/develop/foreman
+    buildargs:
+      hostUID: "{{user_id}}"
+  tags:
+    - build
+
+# reset the bundle configs to download gems as specified
+- name: reset bundle configurations in {{foreman_src}}/.bunlde/config
+  file:
+    state: absent
+    path: "{{foreman_src}}/.bundle/config"
+  tags:
+    - install
+
+- name: install npm and gem packages
+  docker_container:
+    volumes: "{{foreman_src}}:/usr/src/app"
+    user: "{{user_id}}:{{user_id}}"
+    cleanup: yes
+    detach: false
+    name: "install_npm_and_gem_foreman"
+    image: "{{image_name}}:{{image_tag}}"
+    command: "{{item}}"
+  loop:
+    - bundle install --retry=10000 --path=/usr/src/app/gems --without=libvirt:jenkins:openid
+    - npm install
+  tags:
+    - install
+
+- name: remove old database container
+  docker_container:
+    name: "{{foreman_database_name}}"
+    state: absent
+    force_kill: yes
+
+- name: run a new postgres database container
+  docker_container:
+    name: "{{foreman_database_name}}"
+    state: started
+    image: "{{database_docker_images[adapter]}}"
+    env:
+      POSTGRES_PASSWORD: "{{password}}"
+      POSTGRES_USER: "{{username}}"
+      POSTGRES_DB: "{{database}}"
+  when: adapter == "postgresql"
+  tags:
+    - db_setup
+
+- name: run a new mysql/2 database container
+  fail:
+      msg: 'No support to mysql/2 database right now.'
+  when: adapter in ["mysql", "mysql2"]
+  tags:
+    - db_setup
+
+- name: database setup
+  docker_container:
+    volumes: "{{foreman_src}}:/usr/src/app"
+    user: "{{user_id}}:{{user_id}}"
+    cleanup: yes
+    detach: false
+    name: "database_setup"
+    links: "{{None if adapter == 'sqlite3' else foreman_database_name + ':' + db_host}}"
+    image: "{{image_name}}:{{image_tag}}"
+    command: "{{item}}"
+  loop:
+    - bundle exec rake db:migrate
+    - bundle exec rake db:seed
+    - bundle exec rake permissions:reset
```

Comment:
Why is this needed? Seeding should already set up a user + password. You also probably want to make sure you set the env vars to get an initial installation. See https://github.com/theforeman/puppet-foreman/blob/127925156119ec4a76bbb33537a2f1168da095ba/manifests/database.pp#L17-L25 for inspiration.